src/main/java/synapseawsconsolelogin/Auth.java

```diff
@@ -316,17 +318,26 @@ public String getAWSSecretKey() {
 		}
 	}
 	
-	private static Properties properties = null;
-
 	public static void initProperties() {
 		if (properties!=null) return;
 		properties = new Properties();
+		
+		String propertyFileName = System.getProperty(PROPERTIES_FILENAME_PARAMETER);
+		if (StringUtils.isEmpty(propertyFileName)) {
+			propertyFileName = System.getenv(PROPERTIES_FILENAME_PARAMETER);
+			
+		}
+		if (StringUtils.isEmpty(propertyFileName)) {
+			propertyFileName = "global.properties";
+			
+		}
```

Comment:
This logic seems to set the properties load order (lowest to highest) as..
1. system properties 
2. environment variables. 
3. global.properties

However what we actually want is for ENV Vars to override values in properties files.  The `global.properties` should be the fallback so how about change to this load order (lowest to highest) ?
1. system properties 
2. global.properties
3. environment variables.

Reply:
This is not the logic that sets priorities on properties load order.  That logic is in lines 360-371, below.  You will see the priority is, highest to lowest: (1) system properties, (2) properties file, (3) environment var's.  The logic you are commenting on is the logic used to find just the *name* of the properties file, and the priority is, highest to lowest, (1) system properties, (2) environment var's.  I can change both to put env var's as top priority.  Will that suffice?

Reply:
yes, we want env vars as the top priority for both.  thanks.

Reply:
done!